src/index.ts

```diff
@@ -267,15 +287,68 @@ function handleChatCommand(message: ZoomOSCMessage) {
             sendToZoom('/zoom/list');
             break;
         case '/test':
-            // console.log("quicktest: /zoom/clearPin");
-            // sendToZoom("/zoom/clearPin");
+            // console.log("quicktest host", params[1]);
+            // sendToZoom("/zoom/userName/makeCoHost", params[1] );
             break;
         default:
             console.log("handleChatMessage Error: Unimplemented Chat Command");
             sendToZoom('/zoom/zoomID/chat', message.zoomID, "Error: Unimplemented Chat Command");
     }
 }
 
+function processSpecial(recipientZoomID: number, params: string[]): boolean {
+    let specialCommand = true;
+    switch (params[0]) {
+        case '/cohost':
+            if (checkPassword(recipientZoomID, params.slice(2))) {
+                //check if person exists
+                const person: PersonState[] = getPeopleFromName(params[1]);
+                if (!person) {
+                    console.log(`processSpecial: Error - User "${params[1]}" does not exist`);
+                    sendToZoom('/zoom/zoomID/chat', recipientZoomID, `processSpecial: Error - User "${params[1]}" does not exist`);
+                } else {
+                    sendToZoom('/zoom/userName/makeCoHost', params[1]);
+                }
+            }
+            break;
+        case '/end':
+            if (!isHost(recipientZoomID)) {
+                sendToZoom('/zoom/zoomID/chat', recipientZoomID, "Error: Only the Host and Co-hosts can issue Chat Commands");
+            } else {
+                if (checkPassword(recipientZoomID, params.slice(1))) {
+                    sendToZoom("/zoom/endMeeting");
+                }
+            }
+            break;
+        default:
+            specialCommand = false;
+    }
+    return (specialCommand);
+}
+
+function checkPassword(recipientZoomID: number, params: string[]): boolean {
+    let passwordValid = false;
+
+    if (commandPassword == DEFAULT_PASSWORD) {
```

Comment:
use `===` instead of `==`

Reply:
I'm not following why there's a DEFAULT_PASSWORD rather than just having it be null/undefined.